src/loadContract.js

```diff
@@ -29,9 +30,11 @@ module.exports = function(abiPath, address, url) {
   // Get contract
   const Contract = new web3.eth.Contract(abi, address)
 
+  const contractName = path.basename(abiPath, path.extname(abiPath)).split('.')[0]
```

Comment:
If we do the `.split('.')[0]` thing, the `path.extname(abiPath)` argument is not necessary.